aesara/graph/op.py

```diff
@@ -224,8 +224,17 @@ def make_node(self, *inputs: Variable) -> Apply:
                 f"We expected {len(self.itypes)} inputs but got {len(inputs)}."
             )
         if not all(inp.type == it for inp, it in zip(inputs, self.itypes)):
+            n_inputs = len(inputs)
```

Comment:
Not important, but, if this is only used once below, we might as well call `len` where it's used and avoid creating `n_inputs` altogether.